tetris.py

```diff
@@ -27,18 +28,41 @@ def run_gui():
     root = tkinter.Tk()
     root.resizable(False, False)
 
-    topbar = tkinter.Label(root)
-    topbar['bg'] = D_GREY
-    topbar.grid(row=0, column=0, sticky='we')
     game_canvas = tkinter.Canvas(
         root,
         width=square_size * game_width,
         height=square_size * game_height,
         highlightthickness=0,
+        borderwidth=1
     )
-    game_canvas.grid()
+    game_canvas.grid(row=1, sticky='nswe')
 
-    tetris_gui = TetrisGUI(game_speed, game_canvas)
+    topbar = tkinter.Frame(root, bg=D_GREY, height=square_size*2, relief = 'ridge')
+    topbar.grid(row=0, columnspan=2, sticky='we')
+    topbar.columnconfigure(0, weight=1)
```

Comment:
This is not necessary, because you are using `pack` inside topbar, not `grid`.

Reply:
Yes, I've changed that.